pkg/controller/sidecarterminator/vritual_kubelet_handler.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2021 The Kruise Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sidecarterminator
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	AlibabaOrAzureVirtualKubeletKey      = "type"
+	AlibabaOrAzureVirtualKubeletValue    = "virtual-kubelet"
+	AlibabaOrAzureVirtualKubeletTaintKey = "virtual-kubelet.io/provider"
```

Comment:
no need to mention specific vk provider name, just rename to some more general noun, such as DefaultVKTaintKey, DefaultVKLabelKey, DefaultVKLabelValue

Reply:
done